### ISSUE TITLE
vtables: need tableoid in pg_catalog tables

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/cockroach#91113]
Tools like pgadmin need to see the `tableoid` for `pg_catalog` tables for some (unfortunate and sad?) reason. We need to support that too.

**Comment by ajwerner** [comment]
I don't think this is a big deal. We already gave them constant IDs. We can stick with that.

**Comment by jordanlewis** [comment]
Agreed that it's not a big deal! It's just an odd choice by the tools IMHO - that tableoid is the most static thing in the world! People are literally selecting tableoid from pg_class. It's confusing to me.
